### ISSUE TITLE
Remove all break tags <br> and replace with css padding/margin property

### ISSUE THREAD

**Issue by yonikid15** [GCTC-NTGC/TalentCloud#5646]
We need to replace all incorrectly used break tags within the app, since they can [cause problems with screen readers](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/br#accessibility_concerns). Having a break tag within a paragraph tag <p> should be find, however any other uses should be replaced by using the margin/padding property. If the markup is located within a clone block you should use [clones margin and padding property](https://designwithclone.ca/properties/whitespace/). Otherwise you might need to go directly into the sass files and makes changes there.
